kernel32.go

```diff
@@ -461,3 +469,37 @@ func WaitForSingleObject(object HANDLE, msWait uint32) uint32 {
 		return WAIT_FAILED
 	}
 }
+
+func LockFile(hFile HANDLE, fileOffset uint64, numberOfBytesToLock uint64) bool {
+	ret, _, _ := procLockFile.Call(
+		uintptr(hFile),
+		uintptr(fileOffset&0xFFFFFFFF),
+		uintptr(fileOffset>>32),
+		uintptr(numberOfBytesToLock&0xFFFFFFFF),
+		uintptr(numberOfBytesToLock>>32))
+	return ret != 0
+}
+
+const LOCKFILE_EXCLUSIVE_LOCK = 0x00000002
+
+func LockFileEx(hFile HANDLE, fileOffset uint64, numberOfBytesToLock uint64) bool {
```

Comment:
When should `LockFileEx` be used as opposed to `LockFile`? It would be nice to add a comment header to the two functions defining the difference for a caller.